staff/web/vhost/vhosting_check.py

```diff
@@ -43,14 +42,49 @@
     # OCF-Flag.png
     'ce48822dd63785f77d5826b6b215b18d',
     # lighter152x41.gif.png
-    'a9bfe3f918552692f5eddbd3c5446367'}
+    'a9bfe3f918552692f5eddbd3c5446367'} | \
```

Comment:
? why union the two sets? can't you just add a comma there and continue the original set?